scripts/fetch-dna.sh

```diff
@@ -1,2 +1,2 @@
 
-curl -LJ https://github.com/holochain/elemental-chat/releases/download/v0.2.0-alpha20/elemental-chat-skipproof.dna -o ./dnas/elemental-chat.dna
+curl -LJ https://github.com/holochain/elemental-chat/releases/download/v0.2.1-alpha2/elemental-chat.dna -o ./dnas/elemental-chat.dna
```

Comment:
Shouldn't this have skipproof?

Reply:
No file `https://github.com/holochain/elemental-chat/releases/download/v0.2.1-alpha2/elemental-chat-skipproof.dna` exists , but it's working locally with this dna so I think @zo-el probably uploaded a skipproof build

Reply:
Yep. Maybe add a comment?
```suggestion
# Note: DNA specified here must have membrane proofs disabled
curl -LJ https://github.com/holochain/elemental-chat/releases/download/v0.2.1-alpha2/elemental-chat.dna -o ./dnas/elemental-chat.dna
```